test/runtests.jl

```diff
@@ -38,6 +38,19 @@ display(m)
     @test GeoInterface.crs(m) == Tyler.MapTiles.WebMercator()
 end
 
+@testset "NamedTuple axis syntax" begin
+    b = Rect2f(-20.0, -20.0, 40.0, 40.0)
+    m = @test_nowarn Tyler.Map(b, provider=Tyler.TileProviders.OpenStreetMap(), axis = (; type = Axis, aspect = AxisAspect(1)))
+    @test only(contents(m.figure.layout[1, 1])) isa Axis
+    @test only(contents(m.figure.layout[1, 1])).aspect == AxisAspect(1)
```

Comment:
almost, I think here, we need to call-in the value from the `Observable` coming out of `only`. And maybe similarly for the next test as well.

Reply:
True... not sure how i missed that!